controllers/messages.js

```diff
@@ -39,6 +39,8 @@ const getTaskAndDocForMessage = function (messageId, docs) {
       return [task, doc._id];
     }
   }
+
+  return [null,null];
```

Comment:
(Interesting aside: if you don't return `{}` in this instance you actually get a slightly nicer error message too: `Cannot match against 'undefined' or 'null'`. That error is at least understandable)